Content.Client/GameObjects/Components/Power/ApcBoundUserInterface.cs

```diff
@@ -23,48 +23,18 @@ protected override void Open()
         {
             base.Open();
 
-            _window = new SS14Window(IoCManager.Resolve<IDisplayManager>())
+            _window = new ApcWindow(IoCManager.Resolve<IDisplayManager>())
             {
                 MarginRight = 426.0f, MarginBottom = 270.0f
             };
             _window.OnClose += Close;
+            _window.AddToScreen();
 
-            var rows = new VBoxContainer("Rows");
-
-            var statusHeader = new Label("StatusHeader") {Text = "Power Status: "};
-            rows.AddChild(statusHeader);
-
-            var breaker = new HBoxContainer("Breaker");
-            var breakerLabel = new Label("Label") {Text = "Main Breaker: "};
-            _breakerButton = new CheckButton();
+            _breakerButton = _window.Contents.GetChild<CheckButton>("Rows/Breaker/Breaker");
```

Comment:
Instead of doing `GetChild`, it is better to make a public property on `ApcWindow` which contains a reference to the control.

Reply:
Done.